README.md

```diff
@@ -151,22 +169,36 @@ token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&ut
   }
   ```
 
+  If you wish to ignore vulnerabilities for a dependency in a `go.mod` file, you must add `exhortignore` as a comment
+  against the dependency in the manifest file.
+  For example:
+  ```
```

Comment:
To add syntax highlighting, you can set the code block type after the third tick mark.  For example:

\```go

Reply:
Thanks @ritz303 . I have applied the changes that you suggested.

According to the definitions in this [yaml](https://github.com/github-linguist/linguist/blob/7ca3799b8b5f1acde1dd7a8dfb7ae849d3dfb4cd/lib/linguist/languages.yml#L2486) file, keyword `text` should be used to highlight the syntax for `go.mod` file and `requirements.txt` file.

Reply:
Ah, good catch!  Thanks for making the updates, LGTM.